### PR TITLE
Milestone 5 styling

### PR DESCRIPTION
**Pull request by Diegodsha**
- Review the design file and identify all the elements
- Implement the styling of the different components based on the design.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,5 +1,22 @@
-@import url('https://fonts.googleapis.com/css2?family=Nunito&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;600&display=swap');
 
 body {
-  font-family: Nunito, sans-serif;
+  background: #f5f6fa;
```
Comment (eliasecasta) [inline src/index.css]: This color is repeated, don't forget about CSS classes and create one for it, please.

File: src/index.css [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: 100vh;
+}
+
+#root {
+  background-color: #f5f6fa;
```
Comment (eliasecasta) [inline src/index.css]: This color is repeated, don't forget about CSS classes and create one for it, please.

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️

Hello @Diegodsha,

Great job with your project, it looks and works perfectly! It is the best bookstore project that I have seen so far, I really wanted to approve it the first time but you missed a really small and simple detail, please fix it to move on, and once again, well done!

- [x] In the stylesheets of the app, make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
  - no repeating of the same rule (Don't forget to use the CSS classes to avoid rules repetition!)

Please correct those issues and send another review request, good luck, and well done!

File: src/index.css (hunk 2 above)
Comment (Diegodsha) [reply src/index.css]: Thanks for the good comments I'll fix that small detail right now :)

**Comment by rukundoeric** [comment]
# Status: 🍾 PROJECT APPROVED 🍾
_Excellent job on this project guys. 🎆 Keep it up._

- ### To Highlight 
  - Follows gitwork-flow.  💯
  - Codebase looks clean and professional. 💯
  - App is functional and available online. 💯 
  - Issues raised from the previous review are all addressed. 💯 
  

- ### To Improve
  - N/A

_Congratulations on the splendid job **@team** 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/),
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/),
Slack: Rukundo Eric**

**Pull request merged**